shotover/Cargo.toml

```diff
@@ -78,3 +78,7 @@ chacha20poly1305 = { version = "0.10.0", features = ["std"] }
 generic-array = { version = "0.14", features = ["serde"] }
 dyn-clone = "1.0.10"
 kafka-protocol = "0.6.0"
+rustls = { version = "0.21.0", features = ["dangerous_configuration"] }
```

Comment:
Might be worth a comment about why we allow dangerous config (e.g. some DBs we support do bad things (tm))